### PR TITLE
Zach upvote downvote

### PR DESCRIPTION
**Pull request by ZachDFK**
check 'em

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -15,6 +15,8 @@
   <tbody>
     <% @posts.each do |post| %>
       <tr>
+        <% @post = post %>
+        <td><%= render 'vote_form'%></td>
```
Comment (ZachDFK) [inline app/views/posts/index.html.erb:19]: I realize how\* dumb this looks lol but it works

Comment (kevincox) [inline app/views/posts/index.html.erb:19]: Ugh, tables are so 2000. But that's fine for now. We can change it when we style it.

**Comment by kevincox** [comment]
LGTM

File: app/models/user.rb [hunk 2]
```diff
@@ -17,8 +17,13 @@ class User < ActiveRecord::Base
   validates :password, confirmation: true, length: {in: 8..256}  
   validates :password_confirmation, presence: true
   
+  acts_as_voter  
   has_secure_password  
   
+
+
+
```
Comment (kevincox) [inline app/models/user.rb:25]: Why all this space?

**Comment by kevincox** [comment]
Actually is it possible to get some tests?

**Comment by ZachDFK** [comment]
sure I can make tests

**Comment by kevincox** [comment]
Thanks!

File: app/models/user.rb (hunk 2 above)
Comment (ZachDFK) [inline app/models/user.rb:25]: I think I removed I'll tidy up in next commit

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -21,7 +21,7 @@
         <td><%= post.contents %></td>
         <td><%= post.user.username %></td>
         <td><%= link_to 'Show', post %></td>
-        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' }, disabled: :true %></td>
```
Comment (ZachDFK) [inline app/views/posts/index.html.erb:24]: oops, forgot I put that there, doesn't work nor does it bug out. Will remove when we go to the haml file

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -55,13 +55,39 @@ def update
...
   def destroy
-    @post.destroy
-    respond_to do |format|
-      format.html { redirect_to posts_url, notice: 'Post was successfully destroyed.' }
-      format.json { head :no_content }
+    if @post.user == current_user
+      @post.destroy
+      respond_to do |format|
+        format.html { redirect_to posts_url, notice: 'Post was successfully destroyed.' }
+        format.json { head :no_content }
```
Comment (kevincox) [inline app/controllers/posts_controller.rb]: Are these JSON responses actually tested?

**Comment by kevincox** [comment]
Just regen schema.rb and :boat:

**Comment by ZachDFK** [comment]
has been megerd

**Pull request closed without merging**